service-loadbalancer/template.cfg

```diff
@@ -82,11 +82,9 @@ frontend httpfrontend
     # condition to determine the backend to be used
 {{range $i, $svc := .services.http}}
     acl url_acl_{{$svc.Name}} path_beg /{{$svc.Name}}
-{{ if $svc.Host }}
-    acl host_acl_{{$svc.Name}} hdr(host) {{$svc.Host}}
+    {{ if $svc.Host }}acl host_acl_{{$svc.Name}} hdr(host) {{$svc.Host}}
```

Comment:
please add a single comment somewhere here, something like: "the style of if/else blocks is meant to preserves the format of the output config file.",  in case some good samaritan decides to undo your work